packages/sdk/src/utils/validators.ts

```diff
@@ -20,9 +20,18 @@ export const booleanSchema = () =>
 export const stringSchema = () =>
   string().strict().typeError('${path} should be a string');
 
+export const pathSchema = () =>
+  string().typeError('${path} should be a string');
```

Comment:
Why can't we reuse `stringSchema` instead of creating a new one?

Reply:
Commit: f102d9059dad903eb40a70e612f1d0c87c56b0c9